### PR TITLE
changed routhes to routes

### PR DESCRIPTION
**Pull request by avrham408**

### CODE REVIEW

File: my_server/app.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import my_server.log
 import logging
 
+
```
Comment (adarmaor) [inline my_server/app.py]: interesting, why did you add a line?

Comment (avrham408) [reply my_server/app.py]: i love you you are the love of my life
